test/integration/main_test.go

```diff
@@ -188,6 +188,30 @@ func TestMappingEventHandling(t *testing.T) {
 			},
 			wantProcessErrSubStr: "failed to validate and enrich",
 		},
+		{
```

Comment:
I think you can just use resource mapping with subscope as the test case. No need to have one resource mapping without subscope and the other resource mapping with subscope.

Reply:
Updated.